src/diy/smol-slimes/smol-slimes-community-cases.md

```diff
@@ -0,0 +1,392 @@
+# Smol Community Cases
+
+```admonish note
+Designs with no PCB called [stacked designs](./smol-hardware.md#schematics). It's not a negative thing, it's approach that avoid need for custom made PCB.
```

Comment:
not all "no pcb builds" are stacked, the default pinning for supermini for example does not necessarily require pcb